### PR TITLE
Fix curl command in sample-app README

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: examples/sample-app/README.md [hunk 1]
```diff
@@ -185,13 +163,34 @@ This section covers how to perform all the steps of building, deploying, and upd
     be added to the docker-registry service list so that it's reachable from other places.
 
 
-9. Confirm the registry is accessible (you may need to run this more than once):
+6. Confirm the registry is accessible (you may need to run this more than once):
 
-        $ curl `osc get service docker-registry --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}" --config=openshift.local.config/master/admin.kubeconfig`
+        $ curl -H 'Accept: application/json' -v `osc get service docker-registry --template="{{ .spec.portalIP }}:{{ with index .spec.ports 0 }}{{ .port }}{{ end }}/v2/" --config=openshift.local.config/master/admin.kubeconfig`
```
Comment (bparees) [inline examples/sample-app/README.md]: add something in e2e test that validates this curl command also. i'm tired of this getting out of sync.

Comment (mfojtik) [inline examples/sample-app/README.md]: @bparees I added /v2/ uri into e2e test, going to run jenkins for that change....

**Comment by mfojtik** [comment]
[test]

File: examples/sample-app/README.md (hunk 1 above)
Comment (bparees) [inline examples/sample-app/README.md]: cool.

**Comment by bparees** [comment]
[merge]

**Pull request merged**